src/hotspot/share/c1/c1_InstructionPrinter.cpp

```diff
@@ -262,7 +262,7 @@ void InstructionPrinter::print_inline_level(BlockBegin* block) {
 
 
 void InstructionPrinter::print_unsafe_op(UnsafeOp* op, const char* name) {
-  output()->print("%s", name);
+  output()->print("%s(", name);
```

Comment:
Do we need a matching ')' somewhere?

Reply:
It's already there (but on the caller side):
```
void InstructionPrinter::do_UnsafeGet(UnsafeGet* x) {
  print_unsafe_op(x, x->is_raw() ? "UnsafeGet (raw)" : "UnsafeGet");
  output()->put(')');
}
```